src/Nethermind/Ethereum2.Bls.Test/YamlNodeExtensions.cs

```diff
@@ -24,29 +24,32 @@ namespace Ethereum2.Bls.Test
 {
     public static class YamlNodeExtensions
     {
-        public static T? Prop<T>(this YamlNode yamlNode, string propertyName) where T : class
+        public static T?[]? ArrayProp<T>(this YamlNode yamlNode, string propertyName) where T : class
         {
             YamlMappingNode? mappingNode = yamlNode as YamlMappingNode;
-            if (mappingNode is null)
-            {
-                return null;
-            }
-            
-            return (T?) Convert.ChangeType((mappingNode[propertyName] as YamlScalarNode)?.Value, typeof(T));
+            var result =
```

Comment:
this seems to be just a code style that the author of the code has the right to keep their way - if they disagree than this code review creates unnecessary tensions between team members instead of helping to find the significant issues - we have a separate #code-style channel on Slack where we can discuss things about code style that are touching multiple commits

Reply:
I am happy for code reviews to include style & syntax. Actually, I think it is a good idea, including even things like spaces at the ends of lines, etc, to have a consistent code base.

A good approach is to include as many standards as possible in things such as an .editorconfig file.